### ISSUE TITLE
False Error on Trace_block, `first run for txIndex XX error: insufficient funds for gas * price + value`

### ISSUE THREAD

**Issue by jwelch-qn** [erigontech/erigon#12092]
#### System information

Erigon version: `erigon/2.60.8`

OS & Version: Ubuntu 22.04

Commit hash: `dec9d20 `

Erigon Command (with flags/config):
```
curl <Erigon_Endpoint> -X POST   -H "Content-Type: application/json"  --data '{"method":"trace_block","params":["0x24A931"],"id":1,"jsonrpc":"2.0"}'
```

Consensus Layer: `Lighthouse`

Consensus Layer Command (with flags/config): N/A

Chain/Network: 
`Ethereum-Holesky`

#### Expected behaviour
Get tracees for each transaction in the block

#### Actual behaviour
Get `first run for txIndex XX error: insufficient funds for gas * price + value` error

Block in question: https://holesky.etherscan.io/txs?block=2402609
 Has two reverted txs in the block, but not seemingly related to the error message.
 The error refers to the tx at index 26 which is https://holesky.etherscan.io/tx/0x8a60c0ffb6ddb4cf88c0b462063d0a4cb2d31403c0b5fa6b2af1faae8b66597e
the explorer shows it was successful.
 
`eth_getBlockByNumber` for `0x24A931` also shows this tx:
```
   {
      "txHash": "0x8a60c0ffb6ddb4cf88c0b462063d0a4cb2d31403c0b5fa6b2af1faae8b66597e",
      "result": {
        "from": "0x0fd5c0c300b9075d62406da142ab4bbaac908aed",
        "gas": "0x5208",
        "gasUsed": "0x5208",
        "to": "0x7566b700c0eaac88b521536f4e4e1b5b9afe6fe1",
        "input": "0x",
        "value": "0xc4fac0be04544",
        "type": "CALL"
      }
```
#### Steps to reproduce the behaviour
curl <Erigon_Endpoint> -X POST   -H "Content-Type: application/json"  --data '{"method":"trace_block","params":["0x24A931"],"id":1,"jsonrpc":"2.0"}'

**Comment by quickchase** [comment]
Same error is happening on Ethereum Mainnet for block `0x140e6e3` (21030627)

```json
{"jsonrpc":"2.0","id":1,"error":{"code":-32000,"message":"first run for txIndex 85 error: insufficient funds for gas * price + value: address 0x0b92619DdE55C0cbf828d32993a7fB004E00c84B have 1414044108792614043 want 1414044806692614043"}}
```

**Comment by racytech** [comment]
I think those 2 are related https://github.com/erigontech/erigon/issues/12242 issues. Looking into that

**Comment by yperbasis** [comment]
Fixed by PR #12559 and will be released in v2.60.10.
